frontend/components/httpsUrlInput.js

```diff
@@ -13,7 +13,7 @@ const HttpsUrlInput = ({ placeholder, ...props }) => (
   <input
     {...props}
     type="url"
-    pattern="https://.+"
+    pattern="https://.+\.\w{2,}"
```

Comment:
Okay, cool. We have some old code enabling paths in the SQS service. Also, we'll want to validate that on the backend.